.github/workflows/rerun-flaky-workflows.yml

```
@@ -0,0 +1,113 @@
+name: Rerun Flaky Workflows
+
+on:
+  workflow_run:
+    workflows: ["K8s-CI"]
```

```suggestion
    workflows: ["K8s-CI", "CNVM-CI"]
```

See https://github.com/elastic/cloudbeat/pull/1706